lib/reporters/mini.js

```diff
@@ -10,6 +11,10 @@ function MiniReporter() {
 		return new MiniReporter();
 	}
 
+	var spinnerDef = spinners.dots;
+	this.spinnerFrames = spinnerDef.frames;
+	this.spinnerInterval = spinnerDef.interval;
```

Comment:
How about using `var dots = require('cli-spinners').dots` above and just referencing `dots.frames`/`dots.interval`?